core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/workflow/LogicalPlan.scala

```diff
@@ -144,6 +146,28 @@ case class LogicalPlan(
       .toMap
   }
 
+  def resolveScanSourceOpFileName(
```

Comment:
this method is not quite straightforward. add documentation.

Reply:
added